config.py

```diff
@@ -1,7 +1,17 @@
+from os import path
+PROJECT_HOME = path.dirname(path.abspath(__file__))
+S3_CONFIG = path.join(PROJECT_HOME, 'config/s3_config.yml')
+RDS_CONFIG = path.join(PROJECT_HOME, 'config/rds_config.yml')
+
 DEBUG = True
 LOGGING_CONFIG = "config/logging/local.conf"
 PORT = 9038
 APP_NAME = "Football_Manager"
-SQLALCHEMY_DATABASE_URI = 'sqlite:///../data/user_reqs.db'
+DB_PATH = path.join(PROJECT_HOME,'data/msia423.db')
+SQLALCHEMY_DATABASE_URI = 'sqlite:///{}'.format(DB_PATH)
+#DATABASE_NAME = 'msia423'
```

Comment:
Do you think it'd be good to have the same database name for both the RDS as well as the SQLite DB? Since you have the RDS_CONFIG, you could create your sqlalchemy_database_uri there too and then you'd have both databases configured in the same location.

Reply:
K